quick-start-gke.md

```diff
@@ -0,0 +1,124 @@
+# Quick start
+
+In this guide we will see how to start the [Initium Platform](https://github.com/nearform/initium-platform) to a GKE cluster and deploy an application to it from a GitHub action using the [Initium CLI](https://github.com/nearform/initium-cli).
+
+## Prerequisites
+
+- A working GKE cluster
+  - you can install the `gcloud` binary and create a cluster with it using the tools provided by `asdf` in `initium-platform` as explained below
+- The Kubernetes Engine API (container.googleapis.com) enabled in GCP
+- The cluster's control plane should be publicly exposed so the CLI can reach it
```

Comment:
Maybe more generically we can say it needs to network accessible? Just so it implicitly covers VPN, Interconnect scenarios